pkg/.bazelrc

```diff
@@ -0,0 +1,3 @@
+common --enable_platform_specific_config=true
+build:windows --shell_executable="C:/Program Files/Git/usr/bin/bash.exe"

```

Comment:
This won't be portable for other people.  Do you need it now that you dropped the other shell test?